knowledge-base/grid-export-image-column-excel.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Exporting Grid's Image Column to Excel in Blazor
+description: Learn how to export an image column from the Telerik Blazor Grid to an Excel file by utilizing the Document Processing Library.
+type: how-to
+page_title: How to Export Grid's Image Column to Excel with Blazor
+slug: grid-export-image-column-excel
+tags: grid, blazor, telerik, export, excel, image, column
+res_type: kb
+ticketid: 1666986, 1662544, 1645206, 1655440
+---
+
+## Environment
+
+<table>
+<tbody>
+<tr>
+<td>Product</td>
+<td>Grid for Blazor, <br />Document Processing Library</td>
+</tr>
+<tr>
+<td>Version</td>
+<td>6.0.0</td>
+</tr>
+</tbody>
+</table>
+
+## Description
+
+I want to export the Telerik Blazor [Grid]({%slug grid-overview%})'s column that displays images to an Excel file. The images are rendered in a [Grid column template]({%slug grid-templates-column%}). How can I include the image column in the Excel export?
+
+## Solution
+
+To export an image column from the Grid to Excel, follow these steps:
+
+1. Handle the Grid's [`OnAfterExport` event]({%slug grid-export-events%}#onafterexport) to capture the Excel export output as a memory stream.
+2. Use the [Document Processing Library](https://docs.telerik.com/devtools/document-processing/introduction) to modify the Excel file similar to the example in the knowledge base article about [custom cell formatting of the exported file with RadSpreadProcessing]({%slug grid-kb-custom-cell-formatting-with-radspreadprocessing%}). The SpreadProcessing library supports [inserting images in the cells](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/shapes-and-images).
+
+For more details on working with images in the Document Processing Library, refer to the [official documentation](https://docs.telerik.com/devtools/document-processing/introduction).
```

Comment:
The link leads to the Document Processing Overview. We can either adjust the text or change the link to match the text.